wiring/src/spark_wiring_ble.cpp

```diff
@@ -671,39 +670,36 @@ size_t BleAdvertisingData::appendAppearance(ble_sig_appearance_t appearance) {
 }
 
 size_t BleAdvertisingData::resize(size_t size) {
-    selfLen_ = std::min(size, (size_t)BLE_MAX_ADV_DATA_LEN);
-    return selfLen_;
+    selfData_.resize(std::min(size, (size_t)BLE_MAX_ADV_DATA_LEN_EXT));
```

Comment:
We should use `BLE_MAX_ADV_DATA_LEN_EXT_CONNECTABLE` here.

Reply:
See above (connectable application)